.gitignore

```diff
@@ -23,6 +23,7 @@ dist
 storybook-static
 dist-ssr
 *.local
+package-lock.json
```

Comment:
Why we still need to add it to .gitignore, since we already removed package-lock.json, ?

Reply:
I think so, we can prevent adding again this file with mistake later :)